src/main/java/com/commercetools/sync/products/utils/ProductReferenceResolutionUtils.java

```diff
@@ -46,8 +47,7 @@
    * Returns an {@link List}&lt;{@link ProductDraft}&gt; consisting of the results of applying the
    * mapping from {@link Product} to {@link ProductDraft} with considering reference resolution.
```

Comment:
Please update the javadocs:
```suggestion
   * mapping from {@link ProductProjection} to {@link ProductDraft} with considering reference resolution.
```

Also do we need to mention we are doing sync based on the staged product ? 🤔

Reply:
f3bccff3f